core/options.js

```diff
@@ -97,6 +97,10 @@ Blockly.Options = function(options) {
   if (hasCss === undefined) {
     hasCss = true;
   }
+  var hasModal = options['modal'];
+  if (hasModal === undefined) {
```

Comment:
Good point, checked the current code and it seems we need something for:
- window.alert
- window.prompt
- window.confirm

I will prepare this PR to support multiple version and the changes could be added in other PR if needed.